container-build-push-3rd-gen/README.md

```diff
@@ -41,6 +42,8 @@ jobs:
 | image-url           | Image url/name without registry.                                       | Required |
 | image-platforms     | Image platforms to build for. Default is 'linux/amd64'.                | Optional |
 | registry-password   | Registry password.                                                     | Required |
+| scout-user          | Dockerhub user for docker scout. Default is empty                      | Optional |
```

Comment:
You mentioned, that this is required? Or will this be skipped if empty?

Reply:
done